.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -690,6 +690,24 @@ def write(x=''):
     shutil.chown(EMAIL_JSON, 'root', DAEMONPHP_GROUP)
     os.chmod(EMAIL_JSON, 0o440)
 
+##############################################################################
+# Write update_settings.json
+
+UPDATE_SETTINGS_JSON = os.path.join(CONFIG_INSTALL_DIR, 'update_settings.json')
+
+if not os.path.isfile(UPDATE_SETTINGS_JSON):
+    config = OrderedDict()
+    config['upgrade_linux_packages'] = False
+    config['linux_packages_upgrade_type'] = "upgrade" #can be dist-upgrade
+    config['autohandle_apt_prompts'] = False
+    #Refer https://stackoverflow.com/questions/33370297/apt-get-update-non-interactive
+    config['autohandle_apt_prompts_option'] = '--force-confold'
```

Comment:
I'm not familiar with this option but if it can avoid cmd/gui prompts then I'm fine with it, I would think exporting the variable `export DEBIAN_FRONTEND=noninteractive` would also force it to not show any prompts

Reply:
Doing `DEBIAN_FRONTEND=noninteractive` does not stop apt from showing prompts on how to deal with conflicts in the config files for packages.